doc/integrator/c2cwsgiutils.rst

```diff
@@ -23,26 +23,24 @@ To have basic authentication of those tools (and on tile generation) you can set
 You can also activate the authentication on GitHub with OAuth2, the configuration is in the
 ``env.project`` file:
 
-* Verify that the ``C2C_AUTH_GITHUB_REPOSITORY`` is correctly configured.
+* Verify that the ``C2C_AUTH_GITHUB_REPOSITORY`` is correctly configured to the project repository.
 * By default, the ``C2C_AUTH_GITHUB_ACCESS_TYPE`` is set to admin then only admins users of the previously
-  defined repository have access.
-* You should ask Camptocamp for a new secret and set it in ``C2C_AUTH_GITHUB_CLIENT_SECRET``.
-  It will be defined in the GitHub organizations settings, on 'Developer settings', 'OAuth Apps',
-  'Geoservicies` application, 'Generate a new client secret',
-  the URL should also be configured in the redirect proxy.
+  defined repository have access, can be ``pull``, ``push`` or ``admin``.
 * Define the ``C2C_AUTH_GITHUB_SECRET`` randomly.
 
+For independent applications:
```

Comment:
I suppose you mean for applications whose code is not hosted on GitHub?

Reply:
No, it's about the GitHub application, I updated the pull request :-)